diff --git a/concord.js b/concord.js
index 4d5f33b..a96d0b5 100644
--- a/concord.js
+++ b/concord.js
@@ -1156,7 +1156,12 @@ function ConcordEvents(root, editor, op, concordInstance) {
 		if(target.is("a")){
 			if(target.attr("href")){
 				event.preventDefault();
-				window.open(target.attr("href"));
+				if (concordInstance.prefs()["hasspeciallinkhandler"]==true) {				 
+                    lh = concordInstance.prefs()["speciallinkhandler"];
+                    lh(target.attr("href"));
+			    } else {
+				    window.open(target.attr("href"));
+				    }
 				}
 			return;
 			}
